git_machete/tests/functional/test_machete.py

```diff
@@ -2642,21 +2643,33 @@ def test_github_sync(self) -> None:
                 .push()
         )
         self.launch_command('discover', '-y')
-        self.launch_command('github', 'sync', '-y')
+        self.repo_sandbox\
```

Comment:
Nit: for consistency with other places, it can wrapped into `(...)` parens; backslashes won't then be needed